### PR TITLE
example/centos: import OVA

### PR DESCRIPTION
**Pull request by supakeen**
Generate reference manifests, initial dedup. Based on #219.

### CODE REVIEW

**Comment by supakeen** [comment]
Updated PR to use kernel version external.

**Comment by supakeen** [comment]
Undrafted now that #219 has landed.

File: example/centos/common/partition-table/x86_64/ova.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+otk.define:
```
[the remark this answers is not in the document]
Comment (supakeen) [reply example/centos/common/partition-table/x86_64/ova.yaml]: Done.

**Review by mvo5 — APPROVED** [review]
Thanks, this looks good now, I personally would squash merge the two commits for a cleaner history but it's fine.

**Pull request merged**